packages/web/docs/src/components/landing-page.tsx

```diff
@@ -76,16 +78,22 @@ export function IndexPage(): ReactElement {
         </StatsList>
         <UltimatePerformanceCards />
         <Pricing />
-        <GetStartedTodaySection />
+        <TeamSection />
+        <GetStartedTodaySection className="mt-6" />
         <EnterpriseFocusedCards />
       </Page>
     </Tooltip.Provider>
   );
 }
 
-function GetStartedTodaySection() {
+function GetStartedTodaySection({ className }: { className?: string }) {
   return (
-    <section className="relative overflow-hidden bg-[#003834] p-12 text-center sm:rounded-3xl sm:p-24">
+    <section
+      className={cn(
+        'relative overflow-hidden bg-[#003834] p-12 text-center sm:rounded-3xl sm:p-24',
```

Comment:
should we move this `bg` color to tailwind theme?

Reply:
This is something like `green-1000` with a slight blue tint. I tried to use `green-1000` instead of it and it looked worse. I _think_ I'm willing to consider this one to be a one-off to avoid bloat the config.